packages/web/src/i18n.ts

```diff
@@ -1,6 +1,6 @@
 import NextI18Next, { I18nProps as i18nProps } from 'next-i18next'
 
-const options = { defaultLanguage: 'en', otherLanguages: ['de', 'es', 'zh'], saveMissing: false }
+const options = { defaultLanguage: 'en', otherLanguages: ['en'], saveMissing: false }
```

Comment:
Oh wait shouldn't this not have any otherLanguages?

Reply:
i did that first but i18n errors out when its not defined